docs/source/notebooks/DiCE_getting_started.ipynb

```diff
@@ -4,7 +4,40 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Quick introduction to generating counterfactual explanations using DiCE"
+    "# Quick introduction to generating counterfactual explanations using DiCE\n",
+    "\n",
+    "Exploring \"what-if\" scenarios is an important way to inspect a machine learning (ML) model. The DiCE library helps you to understand a ML model by generate \"what-if\" data points that lead to the desired model output. Formally, such \"what-if\" data points are known as *counterfactuals*, described by the following question:\n",
+    "> Given that the model's output for input $x$ is $y$, what would be the output if input $x$ is changed to $x'$?\n",
+    "\n",
+    "The answer to the above question can be obtained by simply inputting $x'$ to the ML model. However, in many cases, we are interested in the reverse question: what changes to $x$ would lead to a desired change in model's output? When inspecting a classifier, for instance, we are often interested in knowing the changes to $x$ that will lead to a desired predicted class. For a regressor, we may be interested in the changes to $x$ that lead to a desired output range. Ideally, these changes should be *minimal* to bring out the local decision logic of the classifier, and *diverse* to show the different ways in which the same outcome can be achieved. The DiCE library provides an easy interface to generate such counterfactual examples for any ML model.\n",
+    "\n",
+    "In addition to proximity (minimal changes) and diversity, another important metric for counterfactual examples is their **feasibility**. If the changes in a counterfactual example are not feasible (e.g., outside the possible range of a particular feature), then the example is less useful. Specifying the possible ranges for continuous features (and possible values for categorical features) is one of the simplest forms of feasibility. DiCE allows you to customize the permitted values for features through the `permitted_range` parameter. In general, feasibility is a complex concept based on whether the [*changes* indicated by a counterfactual are feasible](https://arxiv.org/abs/1912.03277) for the individual, not just that certain values are feasible (e.g., 20 years may be a feasible value for Age feature, but changing a person's age from 22 to 20 is not feasible). DiCE will support such constraints in a future release.\n",
```

Comment:
"DiCE will support such constraints in a future release." - doesn't the VAE method already support feasibility?

Reply:
it does, but currently only works for pytorch.